resources/core/charts/cluster-users/templates/clusterroles-kyma-project-io.yaml

```diff
@@ -0,0 +1,88 @@
+---
+#View access to Kyma resources
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: kyma-view-kyma
+  labels:
+    app: kyma
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    rbac.authorization.kyma-project.io/aggregate-to-kyma-view-kyma: "true"
+  annotations:
+    "helm.sh/hook-weight": "0"
+rules:
+- apiGroups:
+    - "applicationconnector.kyma-project.io"
```

Comment:
How about set all kyma apiGroups as a helm variable? This will allow to modify the list in one place when a new one is introduced

Reply:
Good point - I'll extract common groups to a helm value.